development/tools/pkg/dnscollector/wrappers.go

```diff
@@ -0,0 +1,101 @@
+package dnscollector
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	compute "google.golang.org/api/compute/v1"
+	dns "google.golang.org/api/dns/v1"
+)
+
+//ComputeServiceWrapper A wrapper for compute API service connections.
+type ComputeServiceWrapper struct {
+	Context context.Context
+	Compute *compute.Service
+}
+
+//DNSServiceWrapper A wrapper for dns API service connections.
+type DNSServiceWrapper struct {
+	Context context.Context
+	DNS     *dns.Service
+}
+
+func (csw *ComputeServiceWrapper) lookupRegions(project, pattern string) ([]string, error) {
```

Comment:
I'd rather not lookup regions (see my other comment), but you could pass a callback here to do something with region instead of aggregating them into list. Just an idea.

Reply:
Regions lookup will be removed - explicit list is to be provided instead